app/models/foreman_kubevirt/kubevirt.rb

```diff
@@ -150,6 +156,8 @@ def create_vm(args = {})
         volume.type = 'containerDisk'
         volumes << volume
       end
+      verify_booting_from_image_is_possible(args)
```

Comment:
If we'll move the validation to the image_provision block (line 153) , we can remove the check for the existence image_id key.

what do you think ?

Reply:
@masayag , done :)